rust/src/http2/http2.rs

```diff
@@ -144,6 +147,8 @@ pub struct HTTP2Transaction {
     pub escaped: Vec<Vec<u8>>,
     pub req_line: Vec<u8>,
     pub resp_line: Vec<u8>,
+
+    pub doh: Vec<u8>,
```

Comment:
TODO: add comment to explain this field

Reply:
I feel like it needs a longer name, I suppose a comment would help.